.github/workflows/copybara.yaml

```diff
@@ -0,0 +1,22 @@
+# This script create a Pull request when there is push to copybara branch
```

Comment:
File needs license.

See this comment for an example:

https://github.com/googleapis/google-cloudevents/blob/master/.github/workflows/generate.yaml#L1

Reply:
sounds good